pages/docs/reference/annotations.md

```diff
@@ -246,7 +246,8 @@ public @interface AnnWithArrayValue {
 @AnnWithArrayValue("abc", "foo", "bar") class C
 ```
 
-For other arguments that have an array type, you need to use `arrayOf` explicitly:
+For other arguments that have an array type, you need to use the array literal syntax (since Kotlin 1.2) or 
```

Comment:
Please add a subheading before the line " If the `value` argument in Java has an array type..." ("Arrays as annotation parameters" or something like that), so that it would be easier to refer to the new feature.

Reply:
Done.